user-service/config/database.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require("mongoose");
+const uri = "mongodb://localhost:27017/scrapbookUserService"
```

Comment:
is it possible to rename database name like remove service from it or so ?

Reply:
What do you want the name to be? `scrapbookUser`? 

I think it'll be good to keep the word service too. Like srapbookImageService etc. Let me know what you think!

Reply:
Ok, keeping service is also fine.